### PR TITLE
PointCloud-hotfixUpdate

### PR DESCRIPTION
**Pull request by Kerwin-L**
[feat(pointcloud-box-info): Update the position of boxInfoShow](https://github.com/open-mmlab/labelbee/commit/8c77c1703f3c1f846ca9ea74cc3805f198a91bf5)
[feat(pointcloud-sidebar): Update the sort of component](https://github.com/open-mmlab/labelbee/commit/aca084c5afa037631eed00c206a15505727e175d)

### CODE REVIEW

File: packages/lb-utils/src/annotation/ResourceManager.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    const withInRange = (index: number) => index >= 0 && index < imgList.length;
+
+    /**
+     * The previous {size} and the next {size}
+     */
+    for (let i = 1; i <= size; i += 1) {
+      [currentIndex - i, currentIndex + i].forEach((i: number) => {
+        if (withInRange(i)) {
+          // Img
+          if (imgList[i].url) {
```
Comment (Glenfiddish) [inline packages/lb-utils/src/annotation/ResourceManager.ts:82]: resolve url should extract to a fn

File: packages/lb-utils/src/annotation/ResourceManager.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    });
+  }
+
+  public preloadResourceList(urlList: string[]) {
+    urlList.forEach((url) => {
+      this.fetchResource(url);
+    });
+  }
+
+  public quickPreload({
```
Comment (Glenfiddish) [inline packages/lb-utils/src/annotation/ResourceManager.ts:39]: why this fn has no comment and reference

Comment (Kerwin-L) [reply packages/lb-utils/src/annotation/ResourceManager.ts:39]: Becase it is just to quickPreload

File: packages/lb-utils/src/annotation/ResourceManager.ts (hunk 1 above)
Comment (Kerwin-L) [reply packages/lb-utils/src/annotation/ResourceManager.ts:82]: Is the function need to define outside the class ?

**Pull request merged**